libflux/flux-core/src/semantic/flatbuffers/tests.rs

```diff
@@ -495,6 +495,9 @@ fn compare_exprs(
                 }
                 block_len += 1;
             }
+            if let Some(vectorized) = &semantic_fe.vectorized {
+                compare_exprs(&vectorized, fb_fe.vectorized_type(), &fb_fe.vectorized())?;
+            }
```

Comment:
The test case that uses this code generates semantic graph by parsing and analyzing raw Flux text. That won't populate the `vectorized` field unfortunately. Can you somehow make it so we actually test the new code you wrote? I see two ways to do it:
- modify a function expression in the test case before it's serialized such that the `vectorized` field has something in it.
- Add a special test case with a semantic graph function expression created by hand that contains a function expression with a populated `vectorized` field.

Reply:
This has been implemented.